### DISCUSSION TITLE
Switches show up as the same name

### DISCUSSION THREAD

**Discussion by mcarasso** [Jason-Morcos/homebridge-magic-occupancy#36] [Q&A]
In IOS 16 all of the switches show up as the same name. Is there a good way to fix this?

**Comment by Jason-Morcos** [comment]
Duplicate of https://github.com/Jason-Morcos/homebridge-magic-occupancy/issues/31

See the linked workarounds - this is endemic to all Homebridge plugins and not just this one :/
